package.json

```diff
@@ -4,7 +4,7 @@
     "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
     "test-watch": "meteor test --driver-package practicalmeteor:mocha",
     "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha",
-    "lint": "eslint .",
+    "lint": "meteor node node_modules/eslint/bin/eslint.js .",
```

Comment:
I added that as I didn't do a `npm i -g eslint` (intentionally) this ensures that I am using what is installed with `meteor npm i`  Sort of the same pattern as this https://github.com/trajano/meteor-boilerplate/blob/master/package.json#L8